mergify_engine/web/api/queues.py

```diff
@@ -344,7 +345,13 @@ async def repository_queues_configuration(
     if config_file is None:
         return QueuesConfig()
 
-    config = get_mergify_config(config_file)
+    try:
+        config = get_mergify_config(config_file)
+    except InvalidRules:
+        raise fastapi.HTTPException(
+            status_code=409,
```

Comment:
You need to add this return type in the documentation (see decorator).

Reply:
Shouldn't be 422?

Reply:
Indeed 409 is more for a conflict in the request, in this case the request is good, only the ressource/entity has conflict